src/lib/Microsoft.Health.Extensions.Fhir.R4/Service/ResourceManagementService.cs

```diff
@@ -71,36 +69,13 @@ protected async Task<TResource> CreateResourceByIdentityAsync<TResource>(Model.I
 
             propertySetter?.Invoke(resource, identifier);
 
-            // Generate id programatically and issue an update to FHIR service
-            // The resource will be created if the resource with the given id does not already exist.
-            resource.Id = ComputeSha256(identifier);
-            return await FhirService.UpdateResourceAsync<TResource>(resource).ConfigureAwait(false);
+            return await FhirService.CreateResourceAsync(resource).ConfigureAwait(false);

```

Comment:
nit: Can we add a TODO comment with the tracking item to revert this revert?

Reply:
Thanks for the suggestion.   Going to track with a Github issue for now.